ui/pages/Project/components/RnaSeqOutliers.jsx

```diff
@@ -33,16 +36,15 @@ class RnaSeqOutliersGraph extends React.PureComponent {
   }
 
   initPlot = () => {
-    const { data: dataArray, genesById } = this.props
+    const { data: dataArray, genesById, xField, yField = 'pValue' } = this.props
```

Comment:
nit, but can you make 'pValue' a constant?
also, why doesn't `xField` get a default?

Reply:
I edited my comment to remove the part of the question about `xField` because a second later I saw it defined in `RnaSeqResultPage`, I think you made the right call to make `yField` configurable with a default